.circleci/config.yml

```diff
@@ -79,6 +79,7 @@ jobs:
           # Test also generates, but significantly more
           # Compromise: saving before Test
           paths:
+            - /home/circleci/.rustup/toolchains
```

Comment:
If we're going to cache it, the comment in L58 needs to be removed.

Reply:
Now that I think about it, once we change the nightly version, it will invalidate the cache because the cache key is not affected by the nightly version:
```
cargo_cache_key: cargo-cache-{{ checksum "Cargo.lock" }}-{{ checksum "rust-toolchain.toml" }}-v1
```

It shouldn't cause problems though, because cargo will automatically download the new nightly if it's not there. Let me know if you have thoughts about this.